e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -162,6 +168,27 @@ describe("scenarios > search", () => {
         query: { "source-table": ORDERS_ID },
         dataset: true,
       });
+
+      cy.createModerationReview({
+        status: "verified",
+        moderated_item_type: "card",
+        moderated_item_id: ORDERS_COUNT_QUESTION_ID,
+      });
+    });
+
+    describe("rendering `Verified` filter", () => {
+      it("should render if Pro/EE instance", () => {
+        setTokenFeatures("all");
+        cy.visit("/");
+        cy.findByTestId("search-bar-filter-button").click();
+        cy.findByTestId("verified-search-filter").should("exist");
+      });
+      it("should not render if Starter instance", () => {
+        setTokenFeatures("none");
+        cy.visit("/");
+        cy.findByTestId("search-bar-filter-button").click();
+        cy.findByTestId("verified-search-filter").should("not.exist");
+      });
```

Comment:
This test should be extracted out of `describeEE` otherwise, it won't run on open source edition.